testing/test_aux.out

```diff
@@ -106,5 +115,13 @@ port-02 module_config modules
     `'''       `\__   /\
                 ')
 Redacted docs diff
-No report diff
+46c46
```

Comment:
There must not be any of these lines at the end of the .out file, meaning that the daq folder is clean of spurious files and the test files match the result of the report. I can demonstrate at our next meeting the procedure to get this done correctly.

Reply:
I know this already, This is broken because daq is broken!